velox/docs/configs.rst

```diff
@@ -443,6 +448,18 @@ Each query can override the config by setting corresponding query session proper
      - string
      - 10MB
      - Maximum bytes for sort writer in one batch of output. This is to limit the memory usage of sort writer.
+   * - file-preload-threshold
+     -
+     - integer
+     - 8MB
+     - Usually Velox fetches the meta data firstly then fetch the rest of file. But if the file is very small, Velox can fetch the whole file directly to avoid multiple IO requests. 
+       The parameter controls the threshold when whole file is fetched. 
+   * - footer-size-estinated
```

Comment:
typo: estinated

Did you mean footer-size-estimate? Maybe, footer-estimated-size.

Reply:
Let's use  footer-estimated-size